### PR TITLE
Remove unused subs

### LINKED ISSUE #19541: Remove unused subscriptions from code
# Problem

A lot of subscriptions aren't used anymore but still exist in the code. Not known if that impacts performance but they better to be removed.

### PR DESCRIPTION
**Pull request by vkjr**
fixes #19541 

### Summary
Remove subs that aren't used in the project
### Testing notes
Areas that may be affected by cleanup:
- chats
- communities
- contact management
- profile
- pairing

status: ready

### CODE REVIEW

File: src/status_im/subs/chats.cljs [hunk 1]
```diff
@@ -47,43 +44,6 @@
...
- (fn [[{:keys [chats categories]}] [_ community-id chat-id]]
-   (get categories (get-in chats [(string/replace chat-id community-id "") :categoryID]))))
-
-(re-frame/reg-sub
- :chats/community-chat-by-id
- (fn [[_ community-id _]]
-   [(re-frame/subscribe [:communities/community community-id])])
- (fn [[{:keys [chats]}] [_ community-id chat-id]]
-   (get chats (string/replace chat-id community-id ""))))
-
```
Comment (clauxx) [inline src/status_im/subs/chats.cljs]: I'm using this one in [here](https://github.com/status-im/status-mobile/pull/19529/files)

Comment (vkjr) [reply src/status_im/subs/chats.cljs]: Got it, reverted

**Review by J-Son89 — APPROVED** [review]

File: src/status_im/subs/chats.cljs (hunk 1 above)
Comment (clauxx) [reply src/status_im/subs/chats.cljs]: Thanks ☺️

**Review by briansztamfater — APPROVED** [review]
✂️ ✅

**Review by ibrkhalil — APPROVED** [review]
Thanks for taking the initiative to work on this! 🚀

**Comment by pavloburykh** [comment]
@vkjr thanks for the PR. Please take a look at the issue

### ISSUE 1 "No protocol method IDeref.-deref defined for type undefined" when opening community link for the first time

**Steps:** 

1. Login the app
2. Open any community link (for example https://status.app/c/iw6AChwKCE5ldyBjb21tEgVkYXNzZBgJIgcjNDM2MERGAw==#zQ3shibjPWsBFi527PL5hyBWVq9tWjuXdSXwBscTKgEEx3YwM )
3. See if community is opened

**Actual result:** "No protocol method IDeref.-deref defined for type undefined"  error is displayed. After re-opening the app and opening same link again - community is being fetched. 

[Status-debug-logs - 2024-04-08T102324.400.zip](https://github.com/status-im/status-mobile/files/14902273/Status-debug-logs.-.2024-04-08T102324.400.zip)

https://github.com/status-im/status-mobile/assets/97245802/6039bf46-1c18-4767-a414-c470edeebe57


@vkjr also, could you please leave Testing notes in PR description on what should be tested, which areas have been affected, etc. Thank you!

**Review by clauxx — APPROVED** [review]

**Comment by vkjr** [comment]
@pavloburykh , thanks for finding the issue. Should be fixed now

**Comment by pavloburykh** [comment]
thanks @vkjr. Could you please resolve conflicts in src/status_im/subs/communities.cljs and  leave Testing notes in PR description on what should be tested, which areas have been affected, etc. Thank you!

**Comment by vkjr** [comment]
@pavloburykh, done, thank you!

**Comment by pavloburykh** [comment]
@vkjr thank you for the PR. Ready for merge.

**Comment by vkjr** [comment]
@pavloburykh , thank you!

**Pull request merged**